src/styles/home_body.css

```diff
@@ -0,0 +1,41 @@
+@media only screen and (max-width: 720px) and (max-width: 1800px) {
```

Comment:
I believe it does, the page reacted the same after this code, but I can take a look at that.